breadbox/breadbox/crud/types.py

```diff
@@ -503,9 +522,34 @@ def get_dimension_type_labels_by_id(
     return get_dimension_type_metadata_col(db, dimension_type_name, col_name="label")
 
 
+def get_given_id_to_dimension_id_mapping(
```

Comment:
Does this function actually get used somewhere? 

We'd briefly talked about how you realistically only need the `given_id` to perform most joins (not the `dimension_id`). When I saw this defined here, I was curious to see where we still need a mapping like this, but I can't find anywhere that this function is actually used (only one place that it's imported) 🤔 I may just be missing it though.

Reply:
Oh, that's left over from some earlier (failed) changes. I thought I had removed it. 

...I think it gets removed in my other re-org branch for when I moved things into services. Anyway, I'll confirm and yank it if it's not already removed.

Reply:
yes, confirmed this method is gone in the other branch.